style.css

```diff
@@ -27,9 +32,79 @@
   line-height: 20px;
   margin-left: 24px;
   color: #6070ff;
-  margin-top: 66;
 }
 
 a {
   text-decoration: none;
 }
+
+.home {
+  display: flex;
+  flex-direction: column;
+  background-color: #ffff;
+  background-image: url(/image/mobile-background.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-bottom-left-radius: 85px;
+  justify-content: center;
+  height: 84vh;
+}
```

Comment:
- [ ] [OPTIONAL] If you check the spacing between the `header` and your `h1`, it seems too much and that's because of the `justify-content: center`. Removing it and adding a `margin-top` of required spacing as inspected in the Figma design. This helps to make your design looks more carefully planned and aesthetic 🤟